spec/whois.md

```diff
@@ -0,0 +1,86 @@
+## The `whois` Folder
+
+The `did:webs` method defines that a controller MAY place a folder called
+`whois` within the root folder of the `did:webs` web location and place into
+that [signed files](#signed-files) that are [verifiable credentials] (VCs) or
+[verifiable presentations] (VPs) in which the `did:webs` DID is the subject of the
+verifiable credential. This convention enables those that have the DID for the
+controller to retrieve and verify any VCs or VPs the DID controller has
+decided to publish about itself. The intention is that anyone interested in a
+particular `did:webs` DID can see if there is a `whois` web folder, and if so,
+see if it contains any useful (to the resolver) VCs or VPs that might help
+in learning more about the controller of the DID (the subject of the VCs or VPs).
+
+### `whois` Use Case
+
+The following is a use case for this capability. Consider an example of the
+`did:webs` controller being an educational institution that issues "degree"
+verifiable credentials to its graduate students. A graduate from the school
+submits as part of their job application to a company a [verifiable
+presentation] derived from the verifiable credential they received from the
+school. The company receiving the presentation can verify the cryptography of
+the presentation, but can they trust the school that issued the verifiable
+credential? If the school issued the verifiable credential using its `did:webs`
+DID, the company can resolve the DID (included in the verifiable presentation
+from the student), and verify the [[ref: KEL]]. Then it can look in the DID's
+`whois` folder where it might find a couple of useful VCs or VPs with the
+`did:webs` DID as the subject. For example:
+
+- A verifiable presentation derived from a credential issued by [GLEIF]
+  containing the [LEI] of the school.
+  - Since the company knows about GLEIF, they can automate the lookup to get
+    more information about the school -- where it is registered, etc.
+- A verifiable presentation derived from a credential issued by the "Association
+  of Colleges and Universities" for the jurisdiction of the school.
+  - Since they don't know about the Association for that jurisdiction, the company can
+    repeat the process for the issuer of _that_ credential. They might (for
+    example), learn that the Association has in its `did:webs` `whois` folder a
+    verifiable credential issued by the government of the jurisdiction, and
+    the company recognizes that government.
+- A verifiable presentation derived from a credential issued by US News about
+  the school's placement on the [US News Rankings of Colleges and Universities].
+
+Such checks can all be done with a handful of HTTPS requests and the processing
+of the verifiable presentations. The result is an efficient, verifiable
+credential-based, decentralized trust registries.
```

Comment:
"trust registries" --> "trust registry"

Reply:
Updated.  I did mean to use the plural, but it makes for an awkward sentence as you note.  What I was trying to convey was that where a trust registry might be for a single business domain (e.g. legal entities, educational entities, professional accreditations, etc.) this technique could be used for all of the domains in which the entity operates.

Reply:
I'm fine with plural; I just didn't think "an" and "registries" went together. Maybe plural is better.